assets/state-gpu-admission/0300_daemonset.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Deployment，not DaemonSet

Reply:
done by [commit](https://github.com/easystack/gpu-operator/pull/1/commits/cebe69cd815540e0b082adfec9c77544a7616c4b)